src/transformStateWithClones.js

```diff
@@ -6,8 +6,35 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const array = [];
+  const StateCopy = Object.assign({}, state);
```

Comment:
```suggestion
  const stateCopy = Object.assign({}, state);
```
first letter in lowercase for variables